root/vote/VotingIndex.js

```diff
@@ -9,13 +9,73 @@
 
 import Layout from '../layout/index.js';
 
+const subscribedArtistsCondition =
+  '&conditions.9.field=artist&conditions.9.operator=subscribed';
```

Comment:
Small nitpick: I noticed this first constant starts with `&` but doesn't end in it, while the others don't start with `&` but do end in it. 🙂 TBH I'd probably expect none of them to start *or* end in `&` so that you can use them anywhere in a URL, and choose where the `&` (or `?`) is included as they are used.

Reply:
I guess you could change `subscribedArtistsCondition` to match all the others, but I don't really care that much, the code works fine as it is.